clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -593,7 +593,8 @@ bool ContinuationIndenter::mustBreak(const LineState &State) {
         // name.
         !Style.isJavaScript()) ||
        (Current.is(tok::kw_operator) && Previous.isNot(tok::coloncolon))) &&
-      Previous.isNot(tok::kw_template) && CurrentState.BreakBeforeParameter) {
+      Previous.isNot(tok::kw_template) && CurrentState.BreakBeforeParameter &&
```

Comment:
Not your fault, but I'm not trying to read this condition. Can you make it more readable? We typically use a lambda and early `return false`.

Reply:
I tried to simplify the checks.